price_feeder/tasks.py

```diff
@@ -360,12 +362,23 @@ def post_price(self, price_no_precision, info_contracts, task=None, global_manag
 
         # Multiply factor of the using gas price
         calculated_gas_price = using_gas_price * decimal.Decimal(self.options['gas_price_multiply_factor'])
-
+        # is a price re post
+        if last_used_nonce:
+            if 'last_gas_price' in global_manager:
+                calculated_gas_price = decimal.Decimal(global_manager['last_gas_price']) + 1
+            log.info(" Replacing tx price "
+                 "New gas price: [{0:.6f}] "
+                 "Nonce: [{1}] ".format(
+            calculated_gas_price * 10 ** 18,
+            last_used_nonce,
+            ))
+        
         # arguments to pass to tx
         tx_args = info_contracts['price_feed'].tx_arguments(
             gas_limit=gas_limit,
             gas_price=calculated_gas_price * 10 ** 18,
-            required_confs=0
+            required_confs=0,
+            nonce=last_used_nonce
```

Comment:
This beaing None would make the final nonce to be calculated automatically? Maybe a comment would help if so.